opm/simulators/wells/BlackoilWellModelRestart.cpp

```diff
@@ -256,7 +256,7 @@ loadRestartData(const data::Wells&                 rst_wells,
     {
         const auto& well_name = well_state.name(well_index);
 
-        this->loadRestartWellData(well_name, handle_ms_well, phs,
+        this->loadRestartWellData(well_name, handle_ms_well & !well_state.is_permanently_inactive_well(well_name), phs,
```

Comment:
Sorry for not responding in a timely manner.  This single ampersand (`&`, bit-wise "and") should probably be a double ampersand (`&&`, logical "and") here.

Reply:
Yeah. You are right. I did not notice it.  And I will create a PR for this.

Reply:
Thanks!